### DISCUSSION TITLE
GitHub Co-Pilot only suggesting single line of code

### DISCUSSION THREAD

**Discussion by ryadav96-fynd** [community/community#48507] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello,

I'm currently using GitHub Co-Pilot to assist me with my coding, but I've noticed that it only suggests one line of code at a time. 

Has anyone else experienced this issue or knows of any potential solutions? Any insights or feedback would be greatly appreciated.

**Comment by valentinojosh** [comment]
Hi!

To my knowledge GitHub Co-Pilot mostly delivers one line of code at time by default. In order for it to provide more you need to give it a lot of context, even then, it might still just provide the one line at a time.

Some useful types of context are: select a large block of code and invoke Copilot, provide multi-line ToDo comments, use the suggestion feedback button to attempt to see more lines.

Hope this helps!

**Comment by undefinitely** [comment]
See official comment here https://github.com/orgs/community/discussions/40522#discussioncomment-4701470
